app/controllers/katello/api/v2/content_view_filter_rules_controller.rb

```diff
@@ -122,5 +119,28 @@ def process_errata_ids(select_all_params)
         []
       end
     end
+
+    private def create_rules
+      rule_clazz = ContentViewFilter.rule_class_for(@filter)
+
+      @rules = []
+
+      if rule_params[:name]
+        @rules += rule_params[:name].map do |name|
+          rule_clazz.create!(rule_params.except(:name).merge(:filter => @filter, name: name))
+        end
+      end
+
+      if rule_params[:errata_ids]
+        @rules += rule_params[:errata_ids].map do |errata_id|
+          rule_clazz.create!(rule_params.except(:errata_ids)
+            .merge(filter:  @filter, errata_id: errata_id))
+        end
+      end
+
+      if @rules.empty?
+        @rules = [rule_clazz.create!(rule_params.merge(:filter => @filter))]
+      end
```

Comment:
Seems like the multiple calls to `rule_params` will call the method multiple times instead of grabbing a local variable representing the object and working on it.